### PR TITLE
Milestone 4:

### PR DESCRIPTION
**Pull request by blueberry1312**
### Added desktop version:

- Added images of the desktop version.
- Used media query to responsive design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,32 +15,45 @@ <h3>Welcome</h3>
...
+                        <li class="icon"><a href=""><img src="assets/contact/linkedin-icon.svg" alt="linkedin icon"></a></li>
+                        <li class="icon"><a href=""><img src="assets/contact/angellist-icon.svg" alt="angellist icon"></a></li>
+                        <li class="icon"><a href=""><img src="assets/contact/twitter-icon.svg" alt="twitter icon"></a></li>                
+                        <li class="icon"><a href=""><img src="assets/contact/medium-icon.svg" alt="medium icon"></a></li>             
+                        
+                    </ul>
+                    <button type="button">
+                        <img src="assets/img/normal-button.svg" alt="normal button">
+                    </button>
+                </div>
```
Comment (Ibizugbe) [inline index.html:52]: You have done a nice job in making your page responsive😉 I think you have deviated just a little from the design😉 Make a comparison between the design and your app as regards the spacings around the block of test😉 and implement the necessary changes🤗

The design🤗
your_intro

Your project
design_intro

File: index.html [hunk 2]
```diff
@@ -249,6 +274,14 @@ <h1 class="form-text">I'm always interested in hearing about new projects, so if
...
                 <li class="icon"><a href=""><img src="assets/contact/angellist-icon.svg" alt="angellist icon"></a></li>
             </ul>
+            <ul class="contact-desktop">
+                <li class="icon"><a href=""><img src="assets/contact/github-icon.svg" alt="github icon"></a></li>
+                <li class="icon"><a href=""><img src="assets/contact/linkedin-icon.svg" alt="linkedin icon"></a></li>
+                <li class="icon"><a href=""><img src="assets/contact/angellist-icon.svg" alt="angellist icon"></a></li>
+                <li class="icon"><a href=""><img src="assets/contact/twitter-icon.svg" alt="twitter icon"></a></li>                
+                <li class="icon"><a href=""><img src="assets/contact/medium-icon.svg" alt="medium icon"></a></li>              
+                
+            </ul>
```
Comment (Ibizugbe) [inline index.html:284]: for your contact section on the desktop, I think your project also looks a bit different from the design😉 you can do well to make comparisons and implement the changes🤗 don't forget it is a requirement that as we make our page responsive, we should stick to the design😉

your contact
your_contact

the design
design_contact

kindly take note of the spacings, the background objects, and the width of the form😉 I think the form should occupy more than half of the page by the design😉

File: index.html [hunk 3]
```diff
@@ -161,18 +174,24 @@ <h2 class="card-title">
...
+                    <p>
+                        I can help you build a product , feature or website Look through some of my work and experience! If you
+                        like
+                        what you see and have a project you need coded,
+                        don't hestiate to contact me.
+                    </p> 
+                    <button type="button" class="resume-button">Get My Resume
+                    </button>                
+                </div>    
+            </div>
```
Comment (Ibizugbe) [inline index.html:194]: Your about section looks a little bit different from the design for the desktop view. you can consider adding some padding left and right to the block of texts and make up for the difference in spacing😉

your project
your_about

design
design_about

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @blueberry1312 👋, 

68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!🤗

## Highlights✅

- Linters are passing👍
- Project is well documented👍
- Github flow has been used👍


## Required Changes ♻️
_please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
## Hi @blueberry1312 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 👏🏼 👏🏼 👏🏼 

giphy

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good design 💯 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**